lib/ads/interstitial_ad_manager.js

```diff
@@ -197,11 +200,26 @@ shaka.ads.InterstitialAdManager = class {
    * @param {shaka.extern.AdsConfiguration} config
    */
   configure(config) {
-    if (!this.adContainer_ ||
-        this.usingBaseVideo_ != config.supportsMultipleMediaElements) {
+    this.config_ = config;
+  }
+
+  /**
+   * @private
+   */
+  checkSupportForMultipleMediaElements_() {
```

Comment:
I don't know how happy I am that this method is called `checkSupportForMultipleMediaElements_`, but the variable that it mutates is called `usingBaseVideo_`. It seems non-obvious to me.

I can see three possible changes:

1.  Rename the variable. `usingMultipleMediaElements_`?
2. Rename the method. `determineIfUsingBaseVideo_`?
3. Add a JSDoc description mentioning that this method sets `usingBaseVideo_`.

Of those options I feel like 2 is the best, but I could see arguments for 1 or 3.

Reply:
Ok, done! (2nd option)